schema-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaSchemaSerDe.java

```diff
@@ -50,7 +50,7 @@ public abstract class AbstractKafkaSchemaSerDe {
 
   protected static final byte MAGIC_BYTE = 0x0;
   protected static final int idSize = 4;
-  private static int DEFAULT_CACHE_CAPACITY = 1000;
+  protected static final int DEFAULT_CACHE_CAPACITY = 1000;
```

Comment:
Can this be a tunable instead ?

Reply:
Perhaps in the future.  Currently this default size is used by other caches as well.